Frontend/ui-driver/src/MerchantConfigs/NammaYatriPartnerConfig.js

```diff
@@ -0,0 +1,103 @@
+
+export function currentMerchantString(key){
+    var selectedLanguage = JBridge.getKeysInSharedPrefs("LANGUAGE_KEY");
+    switch(selectedLanguage) {
+        case "HI_IN" :
+            return getStringHIValue(key);
+        case "KN_IN" :
+            return getStringKNValue(key);
+        case "TA_IN" :
+            return getStringTAValue(key);
+        default:
+            return getStringENValue(key);
+      }
+}
+
+export function isCurrentMerchantString(key){
+    return ((key in kannadaStrings) && (key in hindiStrings) && (key in englishStrings) && (key in tamilStrings));
+}
+
+export function getMerchantConfig(key){
+    if (key in nammaYatriConfigs){
```

Comment:
searching key like this might not be efficient instead we can use object[key] which will give undefined